README.md

```diff
@@ -336,40 +336,42 @@ dotnet test VRDR.Tests/DeathRecord.Tests.csproj
 ### VRDR.CLI
 This directory contains a sample command line interface app that uses the VRDR library to do a few different things.
 
+NOTE: In all of the below commands you must specify your preferred version of dotnet, either netcoreapp2.1 or netcoreapp3.1, depending on whether you have installed dotnet 2.1 or dotnet 3.1 locally on your system.
```

Comment:
Is there any way we can set default so the currently installed runtime will be used? It seems like a step back to require this extra flag every time.

Reply:
I messed with it a bit and it I think I have found a solution. I set `<TargetFramework>` to netcoreapp3.1 and then set put both frameworks in `<TargetFrameworks>`. This has allowed me to run it on dotnet core 3.1 without a `--framework` flag, however running on dotnet core 2.1 requires a flag. I unfortunately can't make it completely seamless based on what versions the user has installed.